docroot/sites/archaeology/settings.php

```diff
@@ -820,23 +820,20 @@
   'user_name' => 'displayName',
   'auth_source' => 'default-sp',
   'login_link_display_name' => 'Stanford Login',
-  'header_no_cache' => 1,
-  'logout_goto_url' => NULL,
+  'header_no_cache' => TRUE,
   'user_register_original' => 'visitors',
   'register_users' => TRUE,
   'autoenablesaml' => TRUE,
-  'debug' => TRUE,
+  'debug' => FALSE,
   'secure' => FALSE,
   'httponly' => FALSE,
   'role' => [
-    'population' => 'administrator:suAffiliation,=,hsdo:web|administrator:suAffiliation,=,itservices:webservices',
-    'eval_every_time' => FALSE,
+    'population' => 'administrator:eduPersonEntitlement,=,hsdo:web|administrator:eduPersonEntitlement,=,itservices:webservices',
```

Comment:
Are you sure you want to hard wire these settings? Isn't there a UI to input these?

Reply:
There is a UI, i just took the recommendation on https://www.chapterthree.com/blog/how-to-configure-simplesamlphp-for-drupal-8-on-acquia
And these are the only 2 workgroups released right now. if we switch to use your API call thingy, we can probably move this to normal config.

Reply:
hmm.. Okay. That is for future us to worry about then. Thanks for the explanation.